apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/extract/synthetic/SyntheaRifLoadJobIT.java

```diff
@@ -44,6 +45,15 @@
 public final class SyntheaRifLoadJobIT {
   private static final Logger LOGGER = LoggerFactory.getLogger(SyntheaRifLoadJobIT.class);
 
+  /** only need a single instance of the S3 client. */
+  private static AmazonS3 s3Client;
+
+  /** Sets the minio test container. */
+  @BeforeAll
+  public static void setupMinioTestContainer() {
+    s3Client = S3Utilities.createS3Client(new ExtractionOptions("foo"));
+  }
+
```

Comment:
What do you think of making a base class that contains this block and just extending the test classes that need the minio setup with that base? Could avoid repeating this as much. I was considering doing that on the BFD tests when I get to that point. (Then we'd just have somethjing like SyntheaRifLoadJobIT extends MinioSetupTest or something)